src/html.rs

```diff
@@ -426,7 +426,7 @@ test some special html-characters as &lt; &gt; and &amp; but also &quot; and &#x
     #[async_std::test]
     async fn test_get_html_empty() {
         let t = TestContext::new().await;
-        let msg_id = MsgId::new_unset();
+        let msg_id = MsgId::new(100);
```

Comment:
This test is actually wrong, it tries to load HTML for nonexistent message, which is an error. Message ID 100 at least does not throw an error due to using special ID (rusqlite converters have protection against storing special IDs into database), but the test could be better anyway.